src/lib/int.ts

```diff
@@ -53,7 +42,7 @@ export class UInt64 extends CircuitValue {
 
   static NUM_BITS = 64;
 
-  divMod(y: UInt64 | number): [UInt64, UInt64] {
+  divMod(y: UInt64 | number | string): [UInt64, UInt64] {
```

Comment:
this return type is a bit error prone, why not return something like `{ quotient, rest }` instead?

Especially when we do things like `let  [q] =` or `let [, r] = `